build.gradle

```diff
@@ -33,3 +33,16 @@ task generateKeys {
     }
   }
 }
+
+task runit {    
+    doLast{
+        javaexec { 
+            main="-jar";
+            args = [
+                "cas/build/libs/cas.war",
+                "-Xmx256M"
+            ]
+        }
+        println '---- Runs and flys (Maybe crashes...) ! ----'
```

Comment:
Yup. I was guessing that folks would be less inclined to use/run something if it has the possibility of crashing :)